frontend/src/app/services/desk.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Desk } from './../model/desk.model';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+@Injectable()
+export class DeskService {
+
+  url = 'http://localhost:64513/api/desks';
```

Comment:
url: `http://localhost:64513/api/` powinien trafić do `src\environments\environment.ts` w stałych:
``` ts
export const environment = {
  production: false
};
```
A tak naprawdę to powinien tam być url to naszego api `http://office-pathfinder-api.azurewebsites.net/`

Reply:
tylko na razie nie ma tam danych, na razie bym zostawił zmieni się to później,

Reply:
uruchamiając lokalnie nasz backend i tak łączymy się do bazy na Azure, więc nie przeszkadza zrobić to teraz a nie będziemy musieli za każdym razem uruchamiać backendu.
Jedyna sytuacja gdzie warto mieć podpięcie lokalne to, gdy mamy nowszy backend ale nie jest jeszcze na masterze.
Aczkolwiek link i tak powinien tu trafić czy to lokalny czy nie.